### PR TITLE
Cleanup some code

### PR DESCRIPTION
**Pull request by leesf**
# [BUG]/[FEATURE] title

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

**Comment by FMX** [comment]
The user identifier object is for a feature under development. So maybe there is no need to remove it now.
@AngersZhuuuu Can you help to review this pull request?

**Comment by AngersZhuuuu** [comment]
Yea, should not remove userIdentifier

File: worker/src/test/scala/org/apache/celeborn/service/deploy/cluster/ReadWriteTestBase.scala [hunk 1]
```diff
@@ -51,24 +50,22 @@ trait ReadWriteTestBase extends Logging {
...
 
     val STR3 = RandomStringUtils.random(32 * 1024)
     val DATA3 = STR3.getBytes(StandardCharsets.UTF_8)
     val LENGTH3 = DATA3.length
-    val dataSize3 = shuffleClient.mergeData(APP, 1, 0, 0, 0, DATA3, 0, LENGTH3, 1, 1);
 
     val STR4 = RandomStringUtils.random(16 * 1024)
     val DATA4 = STR4.getBytes(StandardCharsets.UTF_8)
     val LENGTH4 = DATA4.length
-    val dataSize4 = shuffleClient.mergeData(APP, 1, 0, 0, 0, DATA4, 0, LENGTH4, 1, 1);
```
Comment (FMX) [inline worker/src/test/scala/org/apache/celeborn/service/deploy/cluster/ReadWriteTestBase.scala:71]: You break the test logic.

**Comment by FMX** [comment]
You can run "./dev/reformat" to solved all format issues before commit.

File: worker/src/test/scala/org/apache/celeborn/service/deploy/cluster/ReadWriteTestBase.scala (hunk 1 above)
Comment (leesf) [reply worker/src/test/scala/org/apache/celeborn/service/deploy/cluster/ReadWriteTestBase.scala:71]: oops, fixed

**Comment by leesf** [comment]
> You can run "./dev/reformat" to solved all format issues before commit.

Done, just forgot to run the script, I think we could put the instruction into `contributing guide`, but looks like we have no such website yet.

**Comment by AngersZhuuuu** [comment]
> > You can run "./dev/reformat" to solved all format issues before commit.
> 
> Done, just forgot to run the script, I think we could put the instruction into `contributing guide`, but looks like we have no such website yet.

Nice suggestion, no website yet, still under construction.

**Comment by leesf** [comment]
> > > You can run "./dev/reformat" to solved all format issues before commit.
> > 
> > 
> > Done, just forgot to run the script, I think we could put the instruction into `contributing guide`, but looks like we have no such website yet.
> 
> Nice suggestion, no website yet, still under construction.

@AngersZhuuuu is this PR ready to merge?

**Review by FMX — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**